Modix.Web/Pages/UserLookup.razor

```diff
@@ -0,0 +1,184 @@
+﻿@page "/userlookup"
+@attribute [Authorize]
+@using Discord;
+@using Modix.Data.Repositories;
+@using Modix.Services.Core;
+@using Modix.Services.Utilities;
+@using Modix.Web.Components
+@using Modix.Web.Models;
+@using Modix.Web.Models.UserLookup;
+@using Modix.Web.Services
+@using MudBlazor
+@using Discord.WebSocket
+@using System.Linq.Expressions;
+@using MudBlazor.Charts
+@using System.Globalization;
+@using Humanizer;
+@using Modix.Web.Models.Common;
+
+<PageTitle>Modix - User Lookup</PageTitle>
+
+<MudContainer Style="background-color:#f5f5f5">
+    <MudText Typo="Typo.h4">
+        User Lookup@(userInformation is null ? null : $" - {userInformation.Username + (userInformation.Discriminator == "0000" ? "" : "#" + userInformation.Discriminator)}")
+    </MudText>
+
+    <div class="mb-6">
+        <AutoComplete T="ModixUser"
+                      SearchFunc="(query) => DiscordHelper.AutoCompleteAsync(query)"
+                      Placeholder="Username or ID"
+                      SelectedItemChanged="SelectedUserChanged"
+                      Title="Tell us their username"
```

Comment:
Wondering if the "Tell us their username" bit makes it look a little messy/crowded

![image](https://github.com/discord-csharp/MODiX/assets/2829282/4de64163-c93f-4ffc-a986-69abe16692a8)

Reply:
Hm, I guess that's true, I'll make the `Title` param optional